R/new_matrixfit.R

```diff
@@ -389,7 +391,25 @@ new_matrixfit <- function(cf,
     stopifnot(cf$nrObs == 1)
     model_object <- NParticleModel$new(cf$Time, parind, sign.vec, ov.sign.vec)
   }
-  
+
+  if(model == 'n_particles') {
+    initial_guess = function(corr, summands, t1, t2){
```

Comment:
This function should be part of the `NParticleModel` class. Take a look at the base class `MatrixModel`, this contains a function `initial_guess`. You should make this function part of your model.